packages/itwin/grouping-mapping-widget/src/components/Mappings/Import/ConfirmMappingsImport.tsx

```diff
@@ -79,30 +78,39 @@ const ConfirmMappingImport = ({
     setSelectedMappings(newState);
   };
 
+  const importMutation = useMutation({
+    mutationFn: async (selectedMapping: IMappingTyped) => {
+      const accessToken = await getAccessToken();
+      await mappingClient.copyMapping(
+        accessToken,
+        sourceiModelId,
+        selectedMapping.id ?? "",
+        {
+          targetIModelId: iModelId ?? "",
```

Comment:
`targetIModelId` is required. Providing an empty string will result in an error.

Reply:
Left over from incorrect types in the past. Fixed.